frame/nfts/src/features/atomic_swap.rs

```diff
@@ -77,7 +99,15 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		Ok(())
 	}
-
+	/// Cancels the specified swap offer.
+	///
+	/// This function is used to cancel the specified swap offer created by the `caller` account. If
+	/// the swap offer's deadline has not yet passed, the `caller` must be the owner of the offered
+	/// item. After canceling the swap offer, the function emits the `SwapCancelled` event.
```

Comment:
```suggestion
	/// item; otherwise, anyone can cancel an expired offer.
	/// After canceling the swap offer, the function emits the `SwapCancelled` event.
```